client/app/components/login-form.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+const { service } = Ember.inject;
+
+export default Ember.Component.extend({
+  session: service('session'),
```

Comment:
This seems to be done a lot. Would it be better to just inject it through the app stuff instead of the ad-hoc stuff?
https://guides.emberjs.com/v2.2.0/applications/dependency-injection/